metricbeat/module/zookeeper/mntr/data.go

```diff
@@ -0,0 +1,155 @@
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			avgLatency, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_max_latency\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			maxLatency, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_min_latency\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			minLatency, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_packets_received\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			packetsReceived, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_packets_sent\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			packetsSent, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_num_alive_connections\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			numAliveConnections, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_outstanding_requests\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			outstandingRequests, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_znode_count\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			znodeCount, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_watch_count\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			watchCount, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_ephemerals_count\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			ephemeralsCount, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_approximate_data_size\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			approximateDataSize, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_open_file_descriptor_count\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			openFileDescriptorCount, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_max_file_descriptor_count\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			maxFileDescriptorCount, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_followers\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			followers, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_synced_followers\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			syncedFollowers, _ = strconv.Atoi(matches[1])
+		}
+		re = regexp.MustCompile("zk_pending_syncs\\s+(.*$)")
+		if matches := re.FindStringSubmatch(scanner.Text()); matches != nil {
+			pendingSyncs, _ = strconv.Atoi(matches[1])
+		}
+	}
+
+	// TODO: should we remove the zk in front?
```

Comment:
My comment could be better :-)

What I tried to say here is, that the key coming from zookeeper is `zk_znode_count`. But to ES we send `znode_count`. We previously agreed to keep the names as they are. Not sure what we should do here?